data/pirate missions smuggling intro.txt

```diff
@@ -0,0 +1,1217 @@
+			choice
+				`	"What would I be in for?"`
+					goto business
+				`	"If it let me hear more of your voice, I'd eat Zeppadrogon Caviar."`
+			`	Alexandra laughs. "Disgusting, but charming! It's funny you mention that, though..."`
+				goto business
+			
+			label business
+			`	"We've got another pending delivery to the Paradise Worlds. Someone's estate is having a run on some kind of ultra-natural-vego baking goods, and the next scheduled shipment isn't for some time. Head to <stopovers>, pick up the supplies, and drop them off at <destination>."`
+			choice
+				`	"Alright. What's the deadline?"`
+					goto yes
+				`	"I'm sorry, but I don't think I can."`
+			`	Alexandra pauses momentarily. "We really need you on this one, <first>. I wish we had someone else there, but there are no other pilots in the area."`
+			choice
+				`	"Okay, okay. I'll do it. What's the deadline?"`
+					goto yes
+				`	"Nope. I'm taking this one off."`
+			`	"Allow me to reiterate, <first> - this is an ASAP shipment, and heads will roll if it's not delivered immediately. Probably mine, but definitely yours. Your contract will be voided, and Etarip will demand compensation. I'm afraid my hands are tied."`
+			choice
+				`	"Well... Fine, but only because you asked nicely. What's the due date?"`
+					goto yes
+				`	"Could, should, can't, won't. I've got other things going."`
+			`	You hear nothing but silence for a moment. Then, in a flat tone, Alexandra says, "Seems like you're not cut out for this." The transmission ends.`
+				decline
+			
+			label yes
+			`	"<day>. Apparently, the residents of this estate are a having a fit and threatened to personally hunt down any courier who doesn't make it in time. Better get on it!" Alexandra ends the transmission.`
+				accept
+		
+	on decline
+		"rejected etarip" = 3
+	
+	on fail
+		"rejected etarip" = 3
+	
+	on stopover
+		conversation
+			`You land on <stopovers>'s spaceport and oversee the cargo transfer. After exchanging pleasantries with you, the dockworkers begin loading several unmarked crates onto your ship.`
+			`	You notice a pair of Republic security officers watching you closely as you sign the manifest. When you prepare to leave, you see them talking quite energetically to one of the dockworkers who loaded your ship.`
+
+	on complete
+		event "Etarip Intro Finale" 6
+		conversation
+			`Following an uneventful trip, you land on <planet>'s pristine spaceport. A few exceptionally professional dockworkers begin unloading your cargo. You take the opportunity to observe and relax a little. But something feels off...`
+			`	Everything happens at once - you hear a series of shouts and yells as Republic security officers, dressed in full tactical gear, swarm your ship. Several point weapons at you, while others begin breaking into the locked cargo inside and outside your hold.`
+			choice
+				`	[Put your hands up.] "Woah, woah, what's going on here?!?"`

```

Comment:
I feel like another response that leads to the same place might fit here.

Reply:
Agreed! I realized as I was revising it the first time that I had a few "lone" `choice` labels. This is a great place for a second one.